bot/cogs/snakes.py

```diff
@@ -0,0 +1,1208 @@
+                        guess_result.sort()
+                        page_result_list[antidote_tries] = " ".join(guess_result)
+
+                        # Rebuild the board
+                        board = []
+                        for i in range(0, 10):
+                            board.append(f"`{i+1:02d}` "
+                                         f"{page_guess_list[i]} - "
+                                         f"{page_result_list[i]}")
+                            board.append(EMPTY_UNICODE)
+
+                        # Remove Reactions
+                        for emoji in antidote_guess_list:
+                            await board_id.remove_reaction(emoji, user)
+
+                        if antidote_guess_list == antidote_answer:
+                            win = True
+
+                        antidote_tries += 1
+                        guess_result = []
+                        antidote_guess_list = []
+
+                        antidote_embed.clear_fields()
+                        antidote_embed.add_field(name=f"{10 - antidote_tries} "
+                                                      f"guesses remaining",
+                                                 value="\n".join(board))
+                        # Redisplay the board
+                        await board_id.edit(embed=antidote_embed)
+
+        # Winning / Ending Screen
+        if win is True:
+            antidote_embed = Embed(color=SNAKE_COLOR, title="Antidote")
+            antidote_embed.set_author(name=ctx.author.name, icon_url=ctx.author.avatar_url)
+            antidote_embed.set_image(url="https://i.makeagif.com/media/7-12-2015/Cj1pts.gif")
+            antidote_embed.add_field(name=f"You have created the snake antidote!",
+                                     value=f"The solution was: {' '.join(antidote_answer)}\n"
+                                           f"You had {10 - antidote_tries} tries remaining.")
+            await board_id.edit(embed=antidote_embed)
+        else:
+            antidote_embed = Embed(color=SNAKE_COLOR, title="Antidote")
+            antidote_embed.set_author(name=ctx.author.name, icon_url=ctx.author.avatar_url)
+            antidote_embed.set_image(url="https://media.giphy.com/media/ceeN6U57leAhi/giphy.gif")
+            antidote_embed.add_field(name=EMPTY_UNICODE,
+                                     value=f"Sorry you didnt make the antidote in time.\n"
+                                           f"The formula was {' '.join(antidote_answer)}")
+            await board_id.edit(embed=antidote_embed)
+
+        log.debug("Ending pagination and removing all reactions...")
+        await board_id.clear_reactions()
+
+    @command(name="snakes.draw()", aliases=["snakes.draw"])
+    async def draw(self, ctx: Context):
+        """
+        Draws a random snek using Perlin noise
+
+        Written by Momo and kel.
+        Modified by juan and lemon.
+        """
+
+        def beautiful_pastel(hue):
```

Comment:
Same thing here as above, this should probably go in the class instead of the function.